app/main.py

```diff
@@ -14,4 +14,6 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    phrase_lower = phrase.lower()
+    letter_lower = letter.lower()
+    return phrase_lower.count(letter_lower)
```

Comment:
Don't you think creating variables is not nessesary here? Because creating phrase.lower() with prase_lower name doesn't make it more descriptive
So you could remove variable declaration and rewrite using just 1 line